README.md

```
@@ -24,19 +24,24 @@ The easiest way to display the F1 calendar on your E-Ink device is to use our **
+| -------------- | -------------------------------- | ------------------------------------ |
+| `lang`         | `cs` (Czech), `en` (English)     | `?lang=en`                           |
+| `tz`           | Any IANA timezone                | `?tz=America/New_York`               |
+| `year`         | Season year                      | `?year=2026`                         |
+| `round`        | Race round number                | `?year=2026&round=5`                 |
+| `display`      | `1bit`, `spectra6`               | `?display=spectra6`                  |
+| `weather`      | `true`, `false`                  | `?weather=false`                     |
+| `weather_type` | `race_day`, `current`            | `?weather=true&weather_type=current` |
 
 **Examples:**
 
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)